packages/transaction-controller/src/utils/gas-fee-tokens.ts

```diff
@@ -95,6 +96,7 @@ export async function getGasFeeTokens({
           from,
           to,
           value,
+          gas,
```

Comment:
If this is the only route to retrieve the gas fee tokens, how is the `gas` available in the initial request?